omod/src/main/java/org/openmrs/module/dhisreport/web/controller/LocationMappingController.java

```diff
@@ -87,9 +88,28 @@ public void showConfigForm(ModelMap model, WebRequest webRequest) {
 			if (metadata != null) {
 				ou = metadata.getOrganizationUnits().getOrganizationUnits();
 				model.addAttribute("orgunits", ou);
+
+				HashMap<String, String> hm = new HashMap<String, String>();
+				List<Location> locationList = new ArrayList<Location>();
+				locationList.addAll(Context.getLocationService().getAllLocations());
+
+				for (Location l : locationList) {
+					for (LocationAttribute la : l.getActiveAttributes()) {
+						if (la.getAttributeType().getName().equals("CODE")) {
+							if (!la.getValue().toString().isEmpty() && la.getValue().toString() != null) {
```

Comment:
@dkayiwa  I am sorry for the late reply, I didn't see the message. Really Sorry.